bootstrap/digitalocean_k8s_ubuntu_16.04_master.sh

```diff
@@ -23,15 +23,15 @@ apt-get install -y \
 systemctl enable docker
 systemctl start docker
 
-PRIVATEIP=$(ip addr show dev tun0 | awk '/inet / {print $2}' | cut -d"/" -f1)
-echo $PRIVATEIP > /tmp/.ip
-PUBLICIP=$(curl ifconfig.me)
+PUBLICIP=$(curl -s http://169.254.169.254/metadata/v1/interfaces/public/0/ipv4/address)
+VPNIP=$(curl -s http://169.254.169.254/metadata/v1/interfaces/private/0/ipv4/address)
```

Comment:
Same problem here, we'll have to replace metadata with `ip`.

Reply:
Yep, just realized that a second ago #needmorecoffee :) Should be fixed now